.vscode/launch.json

```diff
@@ -26,51 +26,21 @@
       "webRoot": "${workspaceFolder}/app/javascript/packs"
     },
     {
-      "name": "Debug Local File",
-      "type": "Ruby",
-      "request": "launch",
-      "cwd": "${workspaceRoot}",
-      "program": "${workspaceRoot}/main.rb"
-    },
-    {
-      "name": "Listen for rdebug-ide",
-      "type": "Ruby",
-      "request": "attach",
-      "cwd": "${workspaceRoot}",
-      "remoteHost": "127.0.0.1",
-      "remotePort": "1234",
-      "remoteWorkspaceRoot": "${workspaceRoot}"
-    },
-    {
-      "name": "Rails server",
-      "type": "Ruby",
-      "request": "launch",
-      "cwd": "${workspaceRoot}",
-      "program": "${workspaceRoot}/bin/rails",
-      "args": ["server"]
+      "name": "Attach to Existing Server",
+      "type": "ruby_lsp",
+      "request": "attach"
     },
     {
       "name": "RSpec - all",
-      "type": "Ruby",
+      "type": "ruby_lsp",
       "request": "launch",
-      "cwd": "${workspaceRoot}",
       "program": "${workspaceRoot}/bin/rspec",
-      "args": ["-I", "${workspaceRoot}"]
     },
     {
       "name": "RSpec - active spec file only",
-      "type": "Ruby",
-      "request": "launch",
-      "cwd": "${workspaceRoot}",
-      "program": "${workspaceRoot}/bin/rspec",
-      "args": ["-I", "${workspaceRoot}", "${file}"]
-    },
-    {
-      "name": "Cucumber",
```

Comment:
I deleted this Cucumber task since the app no longer uses the Cucumber testing framework.